hack/vendor.sh

```diff
@@ -35,7 +35,7 @@ clone git github.com/coreos/go-etcd v2.0.0
 clone git github.com/hashicorp/consul v0.5.2
 
 # get graph and distribution packages
-clone git github.com/docker/distribution 7dc8d4a26b689bd4892f2f2322dbce0b7119d686
+clone git github.com/docker/distribution ec87e9b6971d831f0eff752ddb54fb64693e51cd
```

Comment:
Can you make a not about the branch this is from? I know we aren't using master since master has the `schema1` package.